AppInspector.CLI/Writers/AnalyzeJsonWriter.cs

```diff
@@ -40,8 +40,8 @@ public override void WriteResults(Result result, CLICommandOptions commandOption
 
             if (cLIAnalyzeCmdOptions.SimpleTagsOnly)
             {
-                List<string> keys = new List<string>(analyzeResult.Metadata.UniqueTags.Keys);
-                keys.Sort();
+                List<string> keys = new List<string>(analyzeResult.Metadata.UniqueTags);
+                //keys.Sort();
```

Comment:
Generally try not to check in commented code

Reply:
Agreed.  Sort no longer needed and should be completed removed -will fix.

Reply:
Isn't UniqueTags a ConcurrentDictionary - e.g. you need .Keys in there somewhere. You should be able to simplify this with:

```
TagsFile tags = new TagsFile() {
  Tags = analyzeResult.Metadata.UniqueTags.Keys.ToArray()
};
```

Reply:
This PR was built based on #246 which has modified the data structures.  Gabe's comments thus are representing both sets of changes.  I have closed PR 251 as requested and submitted a new one based on main to help distinguish the differences better.  Please see #246 first.  Thanks.